src/Prism.Plugin.Popups/Extensions/PageExtensions.cs

```diff
@@ -34,6 +35,9 @@ public static Page GetDisplayedPage(this Page page)
                     case MasterDetailPage mdp:
                         page = mdp.Detail;
                         break;
+                    case FlyoutPage flp:
+                        page = flp.Detail;
+                        break;
```

Comment:
this can actually simply replace MasterDetailPage since it actually inherits from FlyoutPage

Reply:
[replaced](https://github.com/dansiegel/Prism.Plugin.Popups/pull/226/commits/bcd833996395875f9036264e927bb914a66f46ea)